man/systemd.link.xml

```diff
@@ -946,6 +946,21 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MDI=</varname></term>
+        <listitem>
+          <para>Specifies the medium dependent interface (MDI) mode for the interface. A MDI describes
+          the interface from a physical layer implementation to the physical medium used to carry the
+          transmission. Takes one of the following words: <literal>mdi</literal>,
+          <literal>straight</literal>, <literal>mdi-x</literal>, <literal>mdix</literal>,
+          <literal>crossover</literal>, and <literal>auto</literal>. When <literal>mdi</literal> or
+          <literal>straight</literal>, the MDI straight through mode will be used. When
```

Comment:
maybe write this as:

```
…<literal>straight</literal> (or equivalently: <literal>mdi</literal>), <literal>crossover</literal> (or equivalently: <literal>mdi-x</literal>, <literal>mdi</literal>), or <literal>auto</literal>.…
```

i.e. emphasize that these aren't actually six different options, but just three, as three of them are just aliases for two others.